py/selenium/webdriver/remote/webelement.py

```diff
@@ -539,7 +539,10 @@ def location(self):
     @property
     def rect(self):
         """A dictionary with the size and location of the element."""
-        return self._execute(Command.GET_ELEMENT_RECT)['value']
+        if self._w3c:
+            return self._execute(Command.GET_ELEMENT_RECT)['value']
+        else:
+            return self.size, self.location
```

Comment:
This would return a tuple of two dictionaries. You need to combine them and return a dictionary

Reply:
Done, but it might be best not to use `self.size` and `self.location` ? 
I mean may be better to reuse `_execute(Command.GET_ELEMENT_SIZE/LOCATION)`